src/components/Dashboard/Pay.tsx

```diff
@@ -106,6 +113,16 @@ export default function Pay() {
 
   return (
     <div>
+      <div style={{ marginBottom: '0.5rem' }}>
+        {tokenSymbol && tokenAddress && (
+          <TokenAMMPriceBadge
```

Comment:
The idea would be to add a token badge for each DEX/AMM we want to add (next would probs be sushiswap)

Reply:
love it 👏